misc/dist/windows/README.md

```diff
@@ -11,7 +11,7 @@ to extract the Inno Setup installer then run `ISCC.exe` using
 
 ## Building
 
-- Place a Godot editor executable in this folder and rename it to `godot.exe`.
-- Run the Inno Setup Compiler (part of the Inno Setup suite) on the `godot.iss` file.
+- Place a Redot editor executable in this folder and rename it to `redot.exe`.
```

Comment:
Anyone who wants to change the naming convention needs to do it all in one pull request.

These build instructions will be incorrect if this isn't done.